ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2

```diff
@@ -149,6 +149,11 @@ host_monitoring_cron:
   minute: "*/5"
   job: ops-runner -f -s 15 -n csnm.network.int /usr/bin/cron-send-network-metrics
 
+- name: report rpm versions daily
+  hour: "1"
+  minute: "0"
+  job: ops-runner -f -s 120 -n csdov.rpm.versions /usr/bin/cron-send-docker-oc-versions
```

Comment:
What does this do when run against zabbix?

I think we need zabbix items to catch these. Do you agree?